opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/DefaultProvisionService.java

```diff
@@ -1453,4 +1470,22 @@ public void setServiceTypeDao(final ServiceTypeDao dao) {
     public void setEventForwarder(final EventForwarder eventForwarder) {
         m_eventForwarder = eventForwarder;
     }
+
+    public Span buildAndStartSpan(String name, SpanContext spanContext) {
+        if(getTracer() == null) {
```

Comment:
Seems odd to mix calls to `getTracer()` and direct access to `m_tracer` in this same method.